File: examples/js/runtime_error.js
console.log("JavaScript Runtime Error Example");
console.log("This will demonstrate various runtime errors");

// This will cause a ReferenceError
console.log("Trying to access undefined variable:");
console.log(undefinedVariable);  // ReferenceError

// This line won't be reached due to the error above
console.log("This line should not be executed");

// More potential runtime errors (won't be reached):
const obj = null;
console.log(obj.property);  // TypeError

const func = undefined;
func();  // TypeError

File: examples/runtime_error.py
# Example with runtime error
print("Starting the program...")
print("This line will execute fine")

# This will cause a runtime error
try:
    result = 10 / 0  # Division by zero
    print(f"Result: {result}")
except ZeroDivisionError as e:
    print(f"Caught an error: {e}")

print("Program completed!")
